SPECS/libacvp/libacvp.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        A library that implements the client-side of the ACVP protocol
 Name:           libacvp
-Version:        1.2.0
+Version:        1.3.0
```

Comment:
Is there a reason you didn't update all the way up to 1.4.1? Andrew is working on updating our build tooling in `dev` and that's the version, which has all the updates resolving new errors from GCC 10 and 11.

Reply:
Yes, this was intentional. This libacvp 1.3.0 + openssl.patch was used to certify for FIPS and it was recommended to keep the same source baseline. If this version is incompatible with the new GCC toolchain, we'll need to revisit this recommendation and hold on completing this PR.

Reply:
Got it. I just finished checking and it looks like we should be fine with 1.3.0 after all.